src/components/dex/Pair.vue

```diff
@@ -108,6 +106,20 @@ export default {
       return value;
     },
 
+    setMarketData() {
+      this.marketData = _.reduce(this.tickerData, (marketData, market, ticker) => {
```

Comment:
Will `tickerData` always be present?

Reply:
In index there is a default setting. Let me know if that won't handle this not being filled with data. I'm pretty sure the reduce won't error out with an empty object.

Reply:
The default setting will probably cause this to do nothing if nothing if it isn't present. So, I agree this should be fine.